lib/executor.js

```diff
@@ -116,7 +116,9 @@ function setPort(app, instructions) {
     process.env.npm_package_config_port,
     app.get('port'),
     3000
-  ], _.isFinite);
+  ], function(v) {
+    return _.isFinite(parseInt(v, 10));
+  });
```

Comment:
> In order to keep the size of the the browser bundle small, we are trying to get rid of lodash/underscore in as many places as possible

I think that's a worthwhile goal.  However some functions like the String features of `_.capitalize` and `_.camelCase` are IMO worth importing from something like lodash instead of having our own copies which could receive little testing compared to the lodash versions.

To help with the browserify sizing we can move to a pattern like this:
```
var capitalize = require('lodash/string/capitalize');
var camelCase = require('lodash/string/camelCase');
```
Which would allow browserify to only pull in the required functions instead of all of lodash and would make it clearer why this module requires lodash at all.  

@bajtos What do you think of that?